services/httpd/handler.go

```diff
@@ -1191,6 +1193,17 @@ func (h *Handler) servePromRead(w http.ResponseWriter, r *http.Request, user met
 	db := r.FormValue("db")
 	rp := r.FormValue("rp")
 
+	if h.Config.AuthEnabled {
```

Comment:
I agree that the endpoint should be protected with auth. However, this PR could break existing usage where the endpoint is used without auth.

Can you add an option similar to what pprof and ping were given [in this change](https://github.com/influxdata/influxdb/commit/0c4703aaa3cd2cbb245982effd53858cb7443bd9)? I would certainly like to merge this with such an adjustment.